### PR TITLE
feature(StringHelper): remove deprecated util function

### PR DESCRIPTION
**Pull request by dantrau**
@plentymarkets/team-terra

#### Must be done by Terra

Testing

-   [ ] Person 1 (mandatory)
-   [ ] Person 2 (mandatory)

---

#### Must be done by every developer

Please inform a member of Terra (@plentymarkets/team-terra) to get the upper part of the checklist done (with urgency or deadline).

Documentation

-   [ ] Changelog (optional: relevant for every change which influences the API)

Testing

-   [ ] Unit Test (updated or created)

### CODE REVIEW

**Review by zhajjhassan — APPROVED** [review]

File: src/lib/helpers/tests/string.helper.spec.ts [hunk 1]
```diff
@@ -2,20 +2,16 @@ import { StringHelper } from '../string.helper';
 
 describe('StringHelper:', () => {
     it('Method: isNullUndefinedOrEmpty() should validate the variable', () => {
-        // undefined
         let testVar: string = undefined;
         expect(StringHelper.isNullUndefinedOrEmpty(testVar)).toBe(true);
```
Comment (Neill83) [inline src/lib/helpers/tests/string.helper.spec.ts:6]:
```suggestion
        expect(StringHelper.isNullUndefinedOrEmpty(testVar)).toBeTrue();
```

Comment (Neill83) [reply src/lib/helpers/tests/string.helper.spec.ts:6]: wenn es die Methoden gibt, kann man die ja bestimmt verwenden, oder? :)

**Review by Neill83 — CHANGES_REQUESTED** [review]
Changelog fehlt

**Review by MFrank84 — CHANGES_REQUESTED** [review]
Changelog fehlt

**Review by Neill83 — APPROVED** [review]

**Review by MFrank84 — APPROVED** [review]

**Review by Neill83 — APPROVED** [review]

**Review by MFrank84 — APPROVED** [review]

**Pull request merged**